autoload/ensime.vim

```diff
@@ -91,8 +91,8 @@ function! ensime#com_en_format_source(args, range) abort
     return s:call_plugin('com_en_format_source', [a:args, a:range])
 endfunction
 
-function! ensime#com_en_declaration(args, range) abort
-    return s:call_plugin('com_en_declaration', [a:args, a:range])
+function! ensime#fun_en_declaration(args, range) abort
+    return s:call_plugin('fun_en_declaration', [a:args, a:range])
```

Comment:
Actually isn't `ensime#com_en_package_decl` what you meant to rename here, not `com_en_declaration`?